ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/README.md

```diff
@@ -60,7 +60,7 @@ BLOCK cmp-adaptiveform-textinput
 
 ## JavaScript Data Attribute Bindings
 
-Apply a `data-cmp-is="adaptiveFormTextInput"` attribute to the wrapper block to enable initialization of the JavaScript component.
+Apply a `data-cmp-is="adaptiveFormPanel"` attribute to the wrapper block to enable initialization of the JavaScript component.
```

Comment:
This is text input, I guess this is typo

Reply:
yes this has been already changed

Reply:
ok i guess, i switched places will fix.